packages/core/src/components/selectable/selectable.tsx

```diff
@@ -68,6 +71,15 @@ export class Selectable {
     this.mutationObserver?.disconnect();
   }
 
+  componentDidRender() {
+    const checkbox = this.host.shadowRoot!.querySelector('input[type="checkbox"]');
```

Comment:
Ken je https://stenciljs.com/docs/templating-jsx#getting-a-reference-to-a-dom-element ?

Reply:
ja die ken ik inmiddels. dat zou wel een optie kunnen zijn ja.